src/pages/EnablePayments/IdologyQuestions.js

```diff
@@ -54,149 +51,118 @@ const defaultProps = {
-                        style={[styles.mh5, styles.mb5, styles.mt5]}
-                        key={question.type}
-                    >
-                        <Text style={[styles.textStrong, styles.mb5]}>{question.prompt}</Text>
-                        <RadioButtons
-                            items={possibleAnswers}
-                            key={questionIndex}
-                            onPress={(answer) => this.chooseAnswer(questionIndex, answer)}
-                        />
-                    </View>
-                </FormScrollView>
-                <FixedFooter>
-                    <FormAlertWithSubmitButton
-                        isAlertVisible={Boolean(errorMessage)}
-                        onSubmit={this.submitAnswers}
-                        onFixTheErrorsLinkPressed={() => {
-                            this.form.scrollTo({y: 0, animated: true});
-                        }}
-                        message={errorMessage}
-                        isLoading={this.props.walletAdditionalDetails.isLoading}
-                        buttonText={this.props.translate('common.saveAndContinue')}
-                        containerStyles={[styles.mh0, styles.mv0, styles.mb0]}
-                    />
-                    <OfflineIndicator containerStyles={[styles.mh5, styles.mb3]} />
-                </FixedFooter>
+        }
+    };
+
+    return (
+        <View style={styles.flex1}>
+            <View style={styles.ph5}>
+                <Text style={styles.mb3}>{translate('additionalDetailsStep.helpTextIdologyQuestions')}</Text>
+                <TextLink
+                    style={styles.mb3}
+                    href="https://use.expensify.com/usa-patriot-act"
+                >
+                    {translate('additionalDetailsStep.helpLink')}
+                </TextLink>
             </View>
-        );
-    }
+            <FormScrollView ref={formRef}>
+                <View
+                    style={styles.m5}
+                    key={currentQuestion.type}
+                >
+                    <Text style={[styles.textStrong, styles.mb5]}>{currentQuestion.prompt}</Text>
+                    <RadioButtons
+                        items={possibleAnswers}
+                        key={currentQuestionIndex}
+                        onPress={chooseAnswer}
+                    />
+                </View>
+            </FormScrollView>
+            <FixedFooter>
+                <FormAlertWithSubmitButton
+                    isAlertVisible={Boolean(errorMessage)}
+                    onSubmit={submitAnswers}
+                    onFixTheErrorsLinkPressed={() => {
+                        formRef.current.scrollTo({y: 0, animated: true});
+                    }}
+                    message={errorMessage}
+                    isLoading={walletAdditionalDetails.isLoading}
+                    buttonText={translate('common.saveAndContinue')}
+                    containerStyles={[styles.mh0, styles.mb0]}
```

Comment:
Clarification: Why the `styles.mv0` was removed

Reply:
this item doesn't exist in styles object.

Reply:
I;ve added it to the styles

Reply:
Strange, there are three usages of it

Reply:
I guess it can missed while migration to the TS